backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoiceCandidatesAggrgation.feature

```diff
@@ -0,0 +1,221 @@
+@from:cucumber
+Feature: invoice generation and invoice candidates aggregation
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+
+  @from:cucumber
+  Scenario: sales orders with the same salesRep_IDs, one invoice with SalesRep_ID
+    Given metasfresh has date and time 2021-04-16T13:30:13+01:00[Europe/Berlin]
+    And metasfresh contains M_Products:
+      | Identifier | Name            |
+      | p_1        | salesProduct_70 |
+      | p_2        | salesProduct_80 |
```

Comment:
to avoid collision with other tests pls add append a unique suffix to the product name, i.e. the metasfresh/metasfresh issue number